pkg/config/policy.go

```diff
@@ -1,9 +1,28 @@
 package config
 
+import "fmt"
+
 type PolicyWrapper struct {
 	Policies []*Policy `hcl:"policy,block"`
 }
 
+// Normalize normalizes all contained policies.
+func (pw *PolicyWrapper) Normalize() {
+	for _, p := range pw.Policies {
+		p.Normalize()
+	}
+}
+
+// Validate validates all contained policies.
+func (pw *PolicyWrapper) Validate() error {
+	for i, p := range pw.Policies {
+		if err := p.Validate(); err != nil {
+			return fmt.Errorf("policy %s (%d): %w", p.Name, i, err)
+		}
+	}
+	return nil
+}
+
```

Comment:
I would put this logic in the policy decode parser, see config_parser.go and config.go, when parsing and decoding we can have logic when decoding the block see provider.go

Reply:
Done